jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,403 @@
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Order a Basemap Using the Orders API & SDK via AOI & Quad ID"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview ##\n",
+    "---\n",
+    "Download Basemap quads based on the mosaic name, merged and clipped to the area of interest.\n",
+    "\n",
+    "For example, from Explorer, I've defined my area of interest and downloaded that geoJSON.\n",
+    "\n",
+    "I've selected the basemap:\n",
+    "\n",
+    "global_monthly_2022_01_mosaic\n",
+    "\n",
+    "and downloaded my Area of Interest.\n",
+    "\n",
+    "To view the basemap, I clicked on the arrow-in-a-box icon to open that basemap in the Basemap viewer.\n",
+    "\n",
+    "To see all quads in my subject area, I uploaded my Area of Interest.\n",
+    "\n",
+    "326-1257,327-1257,328-1257,326-1256,327-1256,328-1256,326-1255,327-1255,328-1255\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"Get-a-basemap-ID\"></a> \n",
+    "## Get a basemap ID\n",
+    "---\n",
+    "\n",
+    "You can get a basemap ID from Planet Explorer or the Basemap Viewer.\n",
+    "\n",
+    "### Get a basemap ID from Planet Explorer\n",
+    "You&#8217;ll need a basemap ID to identify what basemap to download.\n",
+    "\n",
+    "To pick a basemap, you can open <a href=\"https://www.planet.com/explorer/#\" target=\"_blank\">Planet Explorer</a> and select a basemap from the dropdown menu in the lower left corner. (If you don't have access to basemaps, you'll see a message confirming in the lower right corner. Select \"Get Access\" to sign up for basemaps or speak to your customer service manager.)\n",
+    "\n",
+    "Once you've selected a basemap, the Explorer updates to include that basemap in the view.\n",
+    "\n",
+    "Select the search icon and time cadence, for example monthly or quarterly. Then select one of the resulting basemap IDs, such as \"global_monthly_2022_05_mosaic.\" That's the value you'll be passing into the Orders API. \n",
+    "\n",
+    "### Get a basemap ID from the Basemap Viewer\n",
+    "\n",
+    "To pick a basemap, you can open the Basemap Viewer and select a basemap from the left sidebar menu, using the filter to narrow down the basemap you want.\n",
+    "\n",
+    "Once you've selected a basemap, and selected the right arrow >, the basemap ID displays, such as \"global_monthly_2022_01_mosaic.\" That's the value you'll be passing into the Orders API."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"how-to-get-started\"></a> \n",
+    "## Start coding\n",
+    "\n",
+    "---\n",
+    "Next set up a session by importing needed Python packages, pulling in your API Key, and make an initial request (to retrieve the Orders API parameters) to confirm a connection with the server."
```

Comment:
Instead of calling this section "Start Coding", I would either:
- Include this imports and API key auth stuff near the top of the notebook, as part of the notebook setup, after the overview
OR
- Call the section "Create an Order Packet" and say before we can start working with the API, we need to import some packages and set up our environment